Documentation/CRDs/ceph-cluster-crd.md

```diff
@@ -1334,16 +1334,54 @@ In order to configure an external Ceph cluster with Rook, we need to extract som
 
 1. Run the python script [create-external-cluster-resources.py](/deploy/examples/create-external-cluster-resources.py) for creating all users and keys.
 
+   ```console
+   python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --cephfs-filesystem-name <filesystem-name> --rgw-endpoint  <rgw-endpoint> --namespace <namespace> --format bash
+   ```
+
+- `--namespace`: Namespace where CephCluster will run in the consumer cluster, for example `rook-ceph-external`
+- `--format bash`: The format of the output
+- `--rbd-data-pool-name`: The name of the RBD data pool
+- `--cephfs-filesystem-name`: (optional) The name of the filesystem
+- `--rgw-endpoint`: (optional) The RADOS Gateway endpoint in the format `<IP>:<PORT>`
+- `--rgw-pool-prefix`: (optional) The prefix of the RGW pools. If not specified, the default prefix is `default`
+- `--rgw-tls-cert-path`: (optional) RADOS Gateway endpoint TLS certificate file path
+- `--rgw-skip-tls`: (optional) Ignore TLS certification validation when a self-signed certificate is provided (NOT RECOMMENDED)
+- `--monitoring-endpoint`: (optional) Ceph Manager prometheus exporter endpoints (comma separated list of <IP> entries of active and standby mgrs)
+- `--monitoring-endpoint-port`: (optional) Ceph Manager prometheus exporter port
+- `--ceph-conf`: (optional) Provide a Ceph conf file
+- `--cluster-name`: (optional) Ceph cluster name
+- `--output`: (optional) Output will be stored into the provided file
+- `--cephfs-metadata-pool-name`: (optional) Provides the name of the cephfs metadata pool
+- `--cephfs-data-pool-name`: (optional) Provides the name of the CephFS data pool
+- `--rbd-metadata-ec-pool-name`: (optional) Provides the name of erasure coded RBD metadata pool
+- `--dry-run`: (optional) Prints the executed commands without running them
+- `--rados-namespace`: (optional) Divides a pool into separate logical namespaces
```

Comment:
what about cephfs subvolumegroup?

Reply:
Ntg was introduced for subvolumegroup in external-script and docs,
I will follow up with you on the following issue https://github.com/rook/rook/issues/10427